astro_2.0/features/Proposals/components/ProposalCategoryFilter/ProposalCategoryFilter.module.scss

```diff
@@ -21,7 +21,7 @@
     margin: 0;
     margin-bottom: 32px;
     max-width: initial;
-    top: 168px;
+    top: 176px;
```

Comment:
There is still an issue on the search results page in the "Proposal" tab, I'm afraid.

Reply:
fixed